lib/main.js

```diff
@@ -38,7 +42,18 @@ export default {
       lintOnFly: true,
       lint: (activeEditor) => {
         const command = atom.config.get("linter-ruby.rubyExecutablePath");
+        const ignored = atom.config.get("linter-ruby.ignoredExtensions");
         const filePath = activeEditor.getPath();
+        var ignore = false;
+
+        ignored.split(",").forEach(function (extension) {
```

Comment:
You won't have to split if you make it an array. Also you should do for...of instead of foreach, You won't have to use the `ignore` variable then, you can just return from the loop.